src/bubblewrap.py

```diff
@@ -38,7 +41,9 @@ def __init__(self, txt, bg_color, fg_color, divider, prev_link=None):
 
             # divider fg = prev link bg
             # divider bg = current link bg
-            divider_open, divider_close = get_tags(f"{prev_link.bg_color} on {bg_color}")
+            divider_open, divider_close = get_tags(
+                f"{prev_link.bg_color} on {bg_color}"
+            )
```

Comment:
Not sure why CI check complained about the code I haven't originally touched but formatting the whole file with `black` made it happy.